source/layouts/layout.erb

```diff
@@ -16,7 +16,19 @@
     <link rel="apple-touch-icon" sizes="167x167" href="/assets/images/govuk-apple-touch-icon-167x167.png">
     <link rel="apple-touch-icon" sizes="152x152" href="/assets/images/govuk-apple-touch-icon-152x152.png">
     <link rel="apple-touch-icon" href="/assets/images/govuk-apple-touch-icon.png">
+    <meta property="og:title" content="<%= current_page.data.title && %>">
+    <meta property="og:site_name" content="GOV.UK Pay">
+    <meta property="og:url" content="https://www.payments.service.gov.uk">
+    <meta property="og:description" content="<%= current_page.data.description %>">
+    <meta property="og:type" content="website">
     <meta property="og:image" content="/images/govuk-opengraph-image.png">
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:site" content="@gdsteam" />
+    <meta name="twitter:title" content="<%= current_page.data.title && "#{current_page.data.title} – " %>GOV.UK Pay" />
+    <% if current_page.data.description != '' %>
+    <meta name="twitter:description" content="<%= current_page.data.description %>" />
+    <% end %>
+    <meta name="twitter:image" content="/images/govuk-opengraph-image.png" />
```

Comment:
I can't seem to find `twitter:image` as a valid tag in https://developer.twitter.com/en/docs/twitter-for-websites/cards/guides/getting-started. Is `/images/govuk-opengraph-image.png` meant to be in the repo? It's not at the moment.

Reply:
Well spotted. I'll address it.

Reply:
I'll address the missing image in a separate PR.